lib/realtime/rpc.ex

```diff
@@ -76,4 +77,24 @@ defmodule Realtime.Rpc do
 
       {:error, :rpc_error, reason}
   end
+
+  defp erpc_call(node, mod, func, args, timeout, _opts) when node == node() do
+    :erpc.call(node, mod, func, args, timeout)
+  end
```

Comment:
why exclude local calls from telemetry?

Reply:
Because it will be a normal local call which can be traced normally by the existing spans like here: 

https://github.com/supabase/realtime/blob/basic-opentelemetry/lib/realtime/tenants/connect.ex#L402-L408

This could potentially be handled "transparently" by our RPC module but I'm not sure if it will fit all of our use cases. Let me try something and get back to you :brb: